front/deploygroup.form.php

```diff
@@ -60,8 +60,8 @@
         );
         $values                 = array_pop($item);
 
-        $criteria = ['criteria'     => $_GET['criteria'],
-                        'metacriteria' => $_GET['metacriteria']];
+        $criteria = ['criteria'     =>  isset($_GET['criteria']) ? $_GET['criteria'] : [],
+                     'metacriteria' => isset($_GET['metacriteria']) ? $_GET['metacriteria'] : []];
```

Comment:
I guess same problem may appears on lines 51/52.
You could probably move this declaration before the if/else block to mutualize it and fix both if and else cases.

Reply:
done